### PR TITLE
fix: bump elements and adapt to changes

### PR DESCRIPTION
**Pull request by marcelltoth**
Updates `@stoplight/elements-web-components` and adapts to breaking changes made in stoplightio/elements#615

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -26,7 +26,7 @@
     "@angular/platform-browser": "~10.0.11",
     "@angular/platform-browser-dynamic": "~10.0.11",
     "@angular/router": "~10.0.11",
-    "@stoplight/elements-web-components": "6.0.0-beta.163",
+    "@stoplight/elements-web-components": "^6.0.0-beta.166",
```
Comment (mmiask) [inline package.json:29]: Let's bring it back to `beta`

File: src/app/docs/docs.component.html [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <div class="docs">
   <elements-stoplight-project
-    [workspace]="workspace"
-    [project]="project"
+    [workspaceSlug]="workspaceSlug"
+    [projectSlug]="projectSlug"
```
Comment (mmiask) [inline src/app/docs/docs.component.html:4]: Interesting, I did the exact same thing and it failed on my machine. Nonetheless, thanks for updating that!

**Review by mmiask — APPROVED** [review]
Looks good, only one comment about the dependencies

File: package.json (hunk 1 above)
Comment (mmiask) [reply package.json:29]: Unless there's a reason for a specific version of course

File: src/app/docs/docs.component.html (hunk 2 above)
Comment (marcelltoth) [reply src/app/docs/docs.component.html:4]: Did you also change the corresponding `ts` file?

File: package.json (hunk 1 above)
Comment (marcelltoth) [reply package.json:29]: I'd rather keep it like this, and will set up dependabot to auto-trigger a PR that bumps elements whenever we release a new version. That way we keep the examples up to date immediately and it also runs a sanity check (build) to make sure we didn't break anything.

File: src/app/docs/docs.component.html (hunk 2 above)
Comment (mmiask) [reply src/app/docs/docs.component.html:4]: I did. Error was returned, telling me that /docs could not be found. Although when I pull your branch now, everything seems to work, so maybe I did something different

**Pull request merged**